src/frontend/Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM node:16-alpine AS deps
+FROM node:19-alpine AS deps
```

Comment:
Please use the current LTS version: https://nodejs.org/en/

Reply:
So you mean?

`FROM node:19.1-alpine AS deps`

Reply:
@tommyziegler nope, I meant the LTS version:

>Active LTS - New features, bug fixes, and updates that have been audited by the LTS team and have been determined to be appropriate and stable for the release line.

That would be: 
`FROM node:18-alpine`

Reply:
Ok, have done it now ... Thx. Have a nice weekend